test/completion/stdlib.py

```diff
@@ -158,6 +158,23 @@ def function(a, b):
 #? float()
 tup[1]
 
+class X:
+    def function(self, a, b):
+        return a, b
+    a = functools.partialmethod(function, 0)
+    kw = functools.partialmethod(function, b=1.0)
+
+#? int()
+X().a('')[0]
+#? str()
+X().a('')[1]
```

Comment:
Please test `X.a('')` here as well. Also it would make sense to test `X.a(X(), '')` just to make sure that there are no exceptions (people are probably not going to use it like that. Same for kw below.

Reply:
I think `X.a('')` should throw an exception, so if I should test for that I don't know what comment I should place above. I've only added the `X.a(X(), '')` cases for now.

Reply:
> I think X.a('') should throw an exception

No it shouldn't :) Nothing should ever throw exceptions in Jedi. That case is just there to actually avoid exceptions, I don't really care about the result as long as it doesn't raise exceptions. Obviously it needs to pass tests, but as long as that works it's fine I guess.

Reply:
Yes, you're right, I was thinking of something else. Added the tests now, and they are fine.